pkg/controllers/cassandra/cassandra.go

```diff
@@ -200,6 +213,21 @@ func init() {
 					},
 				),
 			),
+			ctx.SharedInformerFactory.InformerFor(
+				ctx.Namespace,
+				metav1.GroupVersionKind{
+					Version: "v1beta2",
+					Kind:    "StatefulSet",
```

Comment:
Don't forget the group here

Reply:
Done. But note that it doesn't seem to make any difference in my testing.

Reply:
It won't make a difference, but it'll create an extra SharedIndexInformer instead of re-using the existing one, which will cause us to store twice as many StatefulSets in memory 😄 

We should also use v1beta1 here instead of v1beta2, in order to support API servers that don't support v1beta2

Reply:
(BTW - my PR to kubernetes/kubernetes has been updated today, so I'm hoping all this horrible stuff will go away soon 😄 )

Reply:
Ah - just seen you've already updated the version in a later commit!

Reply:
Thanks. In https://github.com/jetstack/navigator/pull/64 I've refactored things a bit so that I can exercise this code in the integration test.